pkg/connect/client/datastore/v1alpha1/fake/fake.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2025 Cofide Limited.
+// SPDX-License-Identifier: Apache-2.0
+
+package fake
+
+import (
+	"context"
+	"sync"
+
+	fakeconnect "github.com/cofide/cofide-api-sdk/pkg/connect/client/fake/connect"
+
+	client "github.com/cofide/cofide-api-sdk/pkg/connect/client/datastore/v1alpha1"
+
+	datastorev1alpha1 "github.com/cofide/cofide-api-sdk/gen/go/proto/connect/datastore_service/v1alpha1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type FakeDataStore struct {
+	Mu            sync.Mutex
+	AttestedNodes map[string]*datastorev1alpha1.AttestedNode
+	NodeSelectors map[string][]*datastorev1alpha1.Selector
+}
```

Comment:
Might be wrong, but I don't think this is really used? Tests only seem to clear the AttestedNodes.